actors/builtin/miner/miner_actor.go

```diff
@@ -1783,19 +1803,53 @@ func validateFRDeclaration(deadlines *Deadlines, deadline *DeadlineInfo, declare
 	return nil
 }
 
-// Computes a fee for a collection of sectors and unlocks it from unvested funds (for burning).
-// The fee computation is a parameter.
-func unlockPenalty(st *State, store adt.Store, currEpoch abi.ChainEpoch, sectors []*SectorOnChainInfo,
-	feeCalc func(info *SectorOnChainInfo) abi.TokenAmount) (abi.TokenAmount, error) {
-	fee := big.Zero()
+// qaPowerForSectors sums the quality adjusted power of all sectors
+func qaPowerForSectors(sectors []*SectorOnChainInfo) (abi.StoragePower, error) {
+	power := big.Zero()
 	for _, s := range sectors {
-		fee = big.Add(fee, feeCalc(s))
+		sectorSize, err := s.Info.SealProof.SectorSize()
```

Comment:
The miner has a single fixed seal proof, and the sectorsize already computed from it at construction and stored in miner info. Use that constant value (pass it as a parameter here). See `PowerForSectors`.

If we ever change that, this will be only one of many places that need updating.